pkgs/applications/misc/dbeaver/default.nix

```diff
@@ -37,7 +37,7 @@ in
     hash = "sha256-TUdtrhQ1JzqZx+QNauNA1P/+WDSSeOGIgGX3SdS0JTI=";
   };
 
-  mvnHash = "sha256-87pf7XRXCuZlAbL54pX+a5Lo/874DmUr/W37/V+5YpQ=";
+  mvnHash = "sha256-ERZYDsPxp1YXteSmunFIgTGZUYqjZJhqrNytLnIUNBQ=";
```

Comment:
(note to self) This package is up-to-date and I opened a PR to enforce all maven plugin versions to be set. With the next update of this package, the hash should no longer be affected by the Maven version.

Reply:
Ok, I got to this repo now and opened the enforce PR: dbeaver/dbeaver#20341

I don't think it's worth waiting for a merge and release, so please disregard w.r.t. #232348